### ISSUE TITLE
[=BUG=] Error 401 Unauthorized

### ISSUE THREAD

**Issue by helloimalastair** [play-dl/play-dl#36]
#### Describe the bug

 - Node.js version: v16.8.0(May vary due to build system, locked to v16 and above)
 - OS & version: Heroku Node Buildpack, latest release

Error 401 (Unauthenticated) occurs after playing a Livestream for an hour.
Livestream is confirmed to be non-copyright, and not age-restricted.

#### Actual behavior
Server Startup, Stream Initiated
Random amount of time passes(around 1 hour, sometimes more, sometimes less)
Server Crash, Error 401 thrown.

#### Expected behavior
Server Startup, Stream Initiated
Stream perpetually runs.

#### Code to reproduce
```js
player = createAudioPlayer({ behaviors: { noSubscriber: NoSubscriberBehavior.Pause } });
const source = await stream(STREAM_URL);
player.play(createAudioResource(source.stream, { inputType: source.type }));
```

#### Checklist

- [x] I have read the documentation.
- [x] I have tried my code with the latest version of Node.js and Play-dl.

**Comment by killer069** [comment]
Please provide url of LiveStream ??

**Comment by helloimalastair** [comment]
> Please provide url of LiveStream ??

https://www.youtube.com/watch?v=5qap5aO4i9A

**Comment by killer069** [comment]
What is the version of play-dl ??

**Comment by helloimalastair** [comment]
> What is the version of play-dl ??
Updated to 0.7.5 earlier today.

**Comment by killer069** [comment]
Okay. Let me do some testing.

**Comment by killer069** [comment]
I didn't found any errors while running that url with [this](https://github.com/play-dl/play-dl/blob/main/examples/play(url).js).

I ran the test for 1.5 hours. But indeed, I found out some issues to tackle with.

I will remain this issue open till I ran into your issue.

**Comment by killer069** [comment]
@helloimalastair 

That error has fixed in 0.7.7

**Comment by helloimalastair** [comment]
> That error has fixed in 0.7.7

Thank you! Got it running now!
